drivers/shared/executor/executor_linux.go

```diff
@@ -37,10 +38,6 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-const (
-	defaultCgroupParent = "/nomad"
-)
```

Comment:
Again, not sure `cgroup_parent` ever really worked apart from the `cpuset` controller.